provision/vvv-init.sh

```diff
@@ -240,6 +240,11 @@ else
   else
     update_wp
   fi
+  cat <<EOF > "${PUBLIC_DIR_PATH}/wp-cli.yml"
+@${VVV_SITE_NAME}:
+  ssh: vagrant@192.168.50.4
+  path: /srv/www/${VVV_SITE_NAME}/public_html
```

Comment:
not every site has this public root or exists in `/srv/www`, since the provisioner is cloned to `vm_dir` we should use that info instead, aka the current working directory, or the folder 1 up from `vvv-iniit.sh`

Reply:
We have a function to get that to avoid duplicate code?

Reply:
VVV's site provisioner passes a constant for the path to the site, and the `public_html` part is defined somewhere in the file as it can come from `config.yml`